docs/source/conf.py

```diff
@@ -307,15 +307,25 @@
     except ImportError:
         # No sphinx_rtd_theme installed
         pass
-    # Loading the dbenv. The backend should be fixed before compiling the
-    # documentation.
-    from aiida.backends.utils import load_dbenv, is_dbenv_loaded
-    if not is_dbenv_loaded():
-        load_dbenv()
+    # Load the database environment by first loading the profile and then loading the backend through the manager
+    from aiida.manage.configuration import get_config, load_profile
+    from aiida.manage.manager import get_manager
+    config = get_config()
+    load_profile(config.default_profile_name)
+    get_manager().get_backend()
 else:
-    # Back-end settings for readthedocs online documentation -
-    # we don't want to create a profile there
-    from aiida.backends import settings
-    settings.IN_DOC_MODE = True
-    settings.BACKEND = "django"
-    settings.AIIDADB_PROFILE = "default"
+    from aiida.manage import configuration
+    from aiida.manage.configuration import load_profile, reset_config
+    from aiida.manage.manager import get_manager
+
+    # Set the global variable to trigger shortcut behavior in `aiida.manager.configuration.load_config`
+    configuration.IN_RT_DOC_MODE = True
+
+    # First need to reset the config, because an empty one will have been loaded when `aiida` module got imported
+    reset_config()
+
+    # Load the profile: this will first load the config, which will be the dummy one for RTD purposes
+    load_profile()
+
+    # Finally load the database backend but without checking the schema because there is no actual database
+    get_manager()._load_backend(schema_check=False)
```

Comment:
Triggering this code path (with some other environment variable?) should be enough to not need the AiiDA DB + profile setup, I think.

Reply:
I think you are right. Do you prefer I do that? I could fake this part by setting `READTHEDOCS=True` in the env vars. Anyway now this (with the latest fix) should work.

Reply:
Yeah, I think it would be a bit neater - hopefully reduces the number of ways things can go wrong since it's the same as on readthedocs.

Reply:
Ok, let's see if with the new changes this now still works... fingers crossed!